modules/openapi-generator/src/main/resources/swift5/swiftformat.mustache

```diff
@@ -0,0 +1,45 @@
+# This file is auto-generated by OpenAPI Generator: https://openapi-generator.tech/
+#
+# For rules on SwiftFormat, please refer to https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md
+#
+# file options
+
+# uncomment below to exclude files, folders
+#--exclude path/to/test1.swift,Snapshots,Build
+
+# format options
+
+--allman false
```

Comment:
@wing328 Hey, how do we decide which rules to keep for this project?
Maybe we should stick to the default one's? 
But if we stick to the default one's, that defeats the purpose of this PR, because the `.swiftformat` would be an empty file...
When we talked in PM I thought that you where asking if we should use SwiftFormat to format the sample projects.
What are your thoughts on this?

Reply:
I used https://github.com/nicklockwood/SwiftFormat/blob/master/.swiftformat as a starting point. Of course users can customize templates to add/delete any rules based on their requirement (as we all know the default won't meet everyone's requirement)

Reply:
Looks good to me 👍